internal/seclang/rule_parser.go

```diff
@@ -173,8 +173,10 @@ func (p *RuleParser) ParseOperator(operator string) error {
 	}
 
 	opRaw, opdataRaw, _ := strings.Cut(operator, " ")
+	opRaw = strings.Clone(opRaw)
```

Comment:
With Go, it's pretty easy to end up with a small string slice referencing a larger document. Not huge, but these did end up retaining the entire line rather than just operator. One part of the investigation is seeing if we don't have anything accidentally retaining the entire ruleset for example.